src/Windowing/Silk.NET.Windowing.Common/Silk.NET.Windowing.Common.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+        <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1</TargetFrameworks>
```

Comment:
Since this is the tutorial target framework, I needed to add this or I would get 'not found' at runtime.